install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -11,144 +11,406 @@
-
-* The owning group of the volume will be set to the specified `*FSGroup*`.
-* Newly created files in the volume will also be owned by `*FSGroup*`.
-* Read and write permissions will be given to `*FSGroup*`.
-* The specified `*FSGroup*` will be added to the pod's list of supplemental
-groups.
-
-This gives your pod, and any other pods with the same `*FSGroup*`, access to the
-volume.
-
-[IMPORTANT]
-====
-Again, it is recommended to allow OpenShift to automatically allocate an
-`*FSGroup*` unless you have a specific use case which requires you to override
-it in the pod definition.
-====
-
-`*FSGroup*` can be specified as follows:
-
-====
-[source,yaml]
-----
-apiVersion: v1
-kind: Pod
+  - name: nfs <1>
+    nfs:
+      server: <nfs-server-ip-or-host>
+      path: /opt/nfs <5>
+```
+<1> name of the volume mount, must match the name in the `volumes` section.
+<2> nfs export path as seen in the container.
+<3> pod global security context: applies to all containers in pod. Note: each container can also define its
+`securityContext`; however, group ids are global to the pod, and cannot be defined for individual containers.
+<4> supplemental groups, which is an array of ids, is set to 5555. This grants group access to the export.
+<5> actual nfs export path on the nfs server.
+
+All containers in the above pod (assuming the matching SCC or project allows the group *5555*) will be
+members of the group *5555*, and will have access to the volume, regardless of the container's user id.
+However, the assumption above is critical. Sometimes, the SCC does not define a range of allowable group
+ids but requires group id validation (due to `supplementalGroups` set to _MustRunAs_; note this is
+not the case for the _restricted_ SCC). And, the namespace will not likely allow a group id of 5555
+(unless the project has been customized for access to this NFS export). So, in this scenario, the above
+pod will fail because its group id of *5555* is not within the SCC's or the namespace's range of allowed
+group ids. 
+
+[[scc-supplemental-groups]]
+==== Supplemental Groups and Custom SCCs
+To remedy this situation a custom SCC can be created such that a min and max group id are defined,
+id range checking is enforced, and the group id of 5555 is allowed. It is considered a better
+practice to create new SCCs versus modifying a predefined SCC, or changing the range of allowed
+ids in the predefined projects. 
+
+The easiest way to create a new SCC is to export an existing SCC and customize the yaml file to 
+meet the requirements of the new SCC. For example:
+```
+# oc export SCC restricted >new-scc.yaml <1>
+##edit new-scc.yaml file
+# oc create -f new-scc.yaml <2>
+```
+<1> use the _restricted_ SCC as a template for the new SCC.
+<2> instantiate the new SCC
+
+*Note:* the `oc edit scc` command can be used to modify an instantiated SCC.
+
+Here is a fragment of a new SCC named "nfs-scc":
+```
+# oc export scc nfs-scc 
```

Comment:
`oc get scc nfs-scc -o yaml` is the better way to read an object like this, `oc export` blanks fields that cannot be set by the user.